Formula/python@3.9.rb

```diff
@@ -92,30 +94,59 @@ class PythonAT39 < Formula
     end
   end
 
+  def xy
+    xy = on_macos do
+      (prefix/"Frameworks/Python.framework/Versions").children.min.basename.to_s if prefix.exist?
+    end
+    xy ||= version.major_minor
+    xy
+  end
```

Comment:
Seeing how ` (prefix/"Frameworks/Python.framework/Versions").children.min.basename.to_s if prefix.exist?` would return 3.9 on my system, do we even still need the `xy`? Or can we just use `#{version.major_minor}`?

Reply:
I think we can use `version.major_minor`, I'm just not 100% certain that this won't break something

Reply:
Me neither, but I think it's worth a try.